api/index.js

```diff
@@ -58,8 +58,9 @@ async function pollForNewNetworks(httpServer, app) {
     )
     console.log(`Transaction service ready at ${config.transactionPath}`)
   })
-  this.newNetworksPollingTimeout = setTimeout(async () => {
-    this.pollForNewNetworks()
+  newNetworksPollingTimeout = setTimeout(async () => {
+    httpServer.close({ port: config.port })
```

Comment:
what?!

Reply:
I was getting an error here. You cannot do `httpServer.listen` without closing the port before, once opened